dev/breeze/src/airflow_breeze/shell/enter_shell.py

```diff
@@ -166,16 +162,14 @@ def build_image_checks(verbose: bool, shell_params: ShellBuilder):
     check_docker_resources(verbose, str(shell_params.airflow_sources), shell_params.airflow_ci_image_name)
     cmd = ['docker-compose', 'run', '--service-ports', '--rm', 'airflow']
     cmd_added = shell_params.command_passed
+    env_variables = construct_env_variables_docker_compose_command(shell_params)
     if cmd_added is not None:
         cmd.extend(['-c', cmd_added])
     if verbose:
-        shell_params.print_badge_info
-        show_cmd = construct_arguments_docker_compose_command(shell_params)
-        show_cmd.extend(cmd)
-        console.print('Command to run:', (' ').join(show_cmd))
-    output = run_command(cmd, verbose=False, text=True)
+        shell_params.print_badge_info()
```

Comment:
Here small correction - print_badge_info is a function not property. I corrected it :)